modules/types_protobuf/proto/CMakeLists.txt

```
@@ -0,0 +1,9 @@
+# =================================================================================================
+# Copyright (C) 2023-2024 HEPHAESTUS Contributors
+# =================================================================================================
+
+set(SOURCES
+    hephaestus/types/proto/dummy_type.proto
+)
+
+define_module_proto_library(NAME types_proto SOURCES ${SOURCES})
```

rule is :`<original_module>_gen_proto`
```suggestion
define_module_proto_library(NAME types_gen_proto SOURCES ${SOURCES})
```

---

Renamed.